evcxr/Cargo.toml

```diff
@@ -52,6 +52,7 @@ ariadne = "0.3.0"
 triomphe = "0.1.8"
 
 pulldown-cmark = { version = "0.9.3", default-features = false, features = ["simd"] }
+tl = "0.7"
```

Comment:
What is the reason to choose this over something like html5ever. It shouldn't matter if a slower html parser is used, right?

Reply:
I didn't choose `tl`, I just know it from a friend yesterday. I not fimalier of `.html` , is there an obvious perfomance gap between the two crates?

Reply:
`tl` claims it is about 10x faster at the cost of not entirely following the html spec. `html5ever` is a lot more popular and does follow the html spec. For this use case I don't think the performance difference should matter, so sticking with `html5ever` may be a better option.

Reply:
Thanks for your advice, I will change the crate dependency and make a new commit.
There is an another difficulty I am struggling at:
The generated `index.html` from report directory has some `<href = some_svg.svg>` hyperlink, which could be clicked at the the showing `.html` and jump to source `some_svg.svg` file. Is there a way I can put the all  these `.svg` files into `index.html`, and then send the `index.html` to jupyter, which the showing shape of `index.html` will be the same as original one?

Reply:
You could encode the `<a href="some_svg.svg">` as `<a href="data:image/svg+xml,base64;AAAA=" target="_blank">` where `AAAA=` is the base64 encoded contents of `some_svg.svg`. I'm not sure if the browser will attempt to show it or download it though. If it attempts to download it you may need to use javascript if jupyter notebook doesn't block that.